spec/features/ach_payment_spec.rb

```diff
@@ -26,7 +25,7 @@
       direction: 'Credit',
       description: 'MyCompany',
       addenda: 'MyCompany monthly payment',
-      idempotency_key: SecureRandom.uuid,
+      idempotency_key: SecureRandom.base64,
```

Comment:
any reason to swap this to base64? (not sure if you were intending to test the factory or the specifying the attribute explicitly)

Reply:
Just to have an explicit set value for testing, since before the implicit value was `nil` :)